### DISCUSSION TITLE
tabby 本地安装插件失败，请问怎么离线安装？

### DISCUSSION THREAD

**Discussion by sunchunfeng** [Eugeny/tabby#9827] [Q&A]
tabby 本地安装插件失败，请问怎么离线安装？
<img width="786" alt="image" src="https://github.com/user-attachments/assets/1ac9416f-e0c4-43e2-a7fa-c258c90cbc71">
本地下载的离线包也解压了，但是重启tabby 软件后还是找不到 sync-config 插件。
<img width="679" alt="image" src="https://github.com/user-attachments/assets/bdfa7924-ff94-4291-9750-98650915ffb3">

**Comment by 0xAllInOne** [comment]
最后怎么解决的？我也碰到同样的问题了。

**Comment by mcmartin666** [comment]
1. 这个插件是Source Code，并未编译，需要自己手动编译后放入插件目录。
2. 插件目录结构如下
`plugins
   |-node_modules
        |-tabby-quick-cmds（在pakcage.json中的name)
            |-dist
            |-package.json`
